### PR TITLE
Version analyzer

### PR DESCRIPTION
**Pull request by RojaEnnam**

### CODE REVIEW

File: Analyzers/Microsoft.Identity.VersionMatchAnalyzer/Microsoft.Identity.VersionMatchAnalyzer.Package/Microsoft.Identity.VersionMatchAnalyzer.Package.csproj [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+  </PropertyGroup>
+
+  <PropertyGroup>
```
Comment (GeoK) [inline Analyzers/Microsoft.Identity.VersionMatchAnalyzer/Microsoft.Identity.VersionMatchAnalyzer.Package/Microsoft.Identity.VersionMatchAnalyzer.Package.csproj:10]: best to pull these from Common.props

File: Analyzers/Microsoft.Identity.VersionMatchAnalyzer/Microsoft.Identity.VersionMatchAnalyzer.Package/Microsoft.Identity.VersionMatchAnalyzer.Package.csproj [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```
Comment (GeoK) [inline Analyzers/Microsoft.Identity.VersionMatchAnalyzer/Microsoft.Identity.VersionMatchAnalyzer.Package/Microsoft.Identity.VersionMatchAnalyzer.Package.csproj:4]: can we also ship net472?

Comment (RojaEnnam) [reply Analyzers/Microsoft.Identity.VersionMatchAnalyzer/Microsoft.Identity.VersionMatchAnalyzer.Package/Microsoft.Identity.VersionMatchAnalyzer.Package.csproj:4]: No, my understanding was netstandard2.0 can work for net472 projects also.

**Comment by jennyf19** [comment]
closing

**Pull request closed without merging**